libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAuthCodeHelper.kt

```diff
@@ -208,15 +204,7 @@ internal class IDPAuthCodeHelper private constructor(
         }
 
         override fun onPageFinished(webView: WebView?, url: String?) {
-            super.onPageFinished(webView, url)
-            if (webView != null) {
-                (webView.parent as? RelativeLayout)?.let { parentView ->
-                    parentView.findViewById<ProgressBar>(R.id.sf__loading_spinner)?.let { spinner ->
-                        spinner.visibility = View.INVISIBLE
-                    }
-                }
-                webView.visibility = View.VISIBLE
-            }
+            wrappedWebViewContent.onPageFinished(webView, url)
```

Comment:
Calling the wrapped WebViewClient - the one defined in the view code which takes care of hiding the spinner and showing the web view